compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/KeywordMap.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.compiler.internal.parser;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class KeywordMap {
+
+    private static final Map<String, Keyword> keywordsMap = Stream.of(Keyword.values()).collect(Collectors
+            .toMap(Keyword::toString, Function.identity()));
+
+    public static Keyword getKeyword(String tokenText) {
+        return keywordsMap.getOrDefault(tokenText, Keyword.DEFAULT);
```

Comment:
If the keyword is not there, Can we fall back to a default language (e.g. english)?

Reply:
Made this change (84391b03d60675b3c2744c4c0fc346c9c5bc2d74) to return english keyword if the foreign language map is not available